mu4/notation/qml/MuseScore/NotationScene/PartsDialog.qml

```diff
@@ -0,0 +1,207 @@
+import QtQuick 2.9
+import QtQuick.Controls 2.12
+import QtQml.Models 2.11
+
+import MuseScore.Ui 1.0
+import MuseScore.UiComponents 1.0
+import MuseScore.NotationScene 1.0
+
+QmlDialog {
+    id: root
+
+    width: 600
+    height: 370
+
+    modal: true
+
+    Rectangle {
+        anchors.fill: parent
+
+        color: ui.theme.popupBackgroundColor
+
+        PartListModel {
+            id: partsModel
+        }
+
+        ItemSelectionModel {
+            id: selectionModel
+            model: partsModel
+        }
+
+        Component.onCompleted: {
+            partsModel.load()
+        }
+
+        Column {
```

Comment:
What happens if the user decides to stretch the window?
I think it's better to use ColumnLayout

Reply:
fixed